src/main.rs

```diff
@@ -82,6 +82,22 @@ lazy_static! {
 #[tokio::main]
 async fn main() -> Result<()> {
     let matches = cli::build_cli().get_matches();
+    let mut term_color_support = "dumb".to_string();
+
+    if let Ok(val) = env::var("TERM") {
+        term_color_support = val
+    }
+
+    let no_color = matches
+        .get_one::<bool>("no-color")
+        .unwrap_or(&false)
+        .to_owned();
+
+    if no_color {
+        env::set_var("TERM", "dumb");
+    } else {
+        env::set_var("TERM", term_color_support);
+    }
```

Comment:
I think this can be dropped now, isn't it?

Reply:
Well no. With prettytable the default is to print with no colors. But the row definitions in the code are something like this: 

```rust
table.add_row(row![Fgbl -> "mutating:", metadata.mutating]);
table.add_row(row![Fgbl -> "background audit support:", metadata.background_audit]);
table.add_row(row![Fgbl -> "context aware:", !metadata.context_aware_resources.is_empty()]);
table.add_row(row![Fgbl -> "execution mode:", metadata.execution_mode]);
```

I couldn't find a global method in prettytable that would disable color. So we would have to rewrite the row definitions  like this:
```rust
if no_color {
  table.add_row(row![Fgbl -> "mutating:", metadata.mutating]);
  table.add_row(row![Fgbl -> "background audit support:", metadata.background_audit]);
  table.add_row(row![Fgbl -> "context aware:", !metadata.context_aware_resources.is_empty()]);
  table.add_row(row![Fgbl -> "execution mode:", metadata.execution_mode]);
}else{
  table.add_row(row!["mutating:", metadata.mutating]);
  table.add_row(row!["background audit support:", metadata.background_audit]);
  table.add_row(row!["context aware:", !metadata.context_aware_resources.is_empty()]);
  table.add_row(row!["execution mode:", metadata.execution_mode]);
}
```

This would cause a lot of redundant code.